$OpenBSD: patch-strigidaemon_CMakeLists_txt,v 1.1 2019/11/23 10:38:35 bket Exp $

Do not pick up cppunit as newer versions of it require C++11 for compiling,
hence any project using cppunit must be compiled with C++11 support. strigi
doesn't build with C++11.

Index: strigidaemon/CMakeLists.txt
--- strigidaemon/CMakeLists.txt.orig
+++ strigidaemon/CMakeLists.txt
@@ -65,7 +65,7 @@ find_optional_dep(FAM ENABLE_FAM FAM_FOUND "efficient 
 find_optional_dep(Log4cxx ENABLE_LOG4CXX LOG4CXX_FOUND "advanced logging")
 #find_optional_dep(CLucene ENABLE_CLUCENE CLucene_FOUND "CLucene backend")
 #find_optional_dep(Exiv2 ENABLE_EXIV2 EXIV2_FOUND "indexing of EXIF/IPTC metadata")
-find_package(CppUnit)
+#find_package(CppUnit)
 include(ConfigureChecks.cmake)
 set (LIBSEARCHCLIENT_VERSION ${STRIGI_VERSION_STRING})
 
